src/common/src/session_config/mod.rs

```diff
@@ -372,6 +381,14 @@ impl ConfigMap {
             self.streaming_enable_delta_join = val.as_slice().try_into()?;
         } else if key.eq_ignore_ascii_case(EnableTwoPhaseAgg::entry_name()) {
             self.enable_two_phase_agg = val.as_slice().try_into()?;
+            if *self.enable_two_phase_agg {
```

Comment:
```suggestion
            if !*self.enable_two_phase_agg {
```

Reply:
I think it should be true?

If two phase agg becomes enabled i.e. `self.enable_two_phase_agg == true`, we should then disable `FORCE_TWO_PHASE_AGG`.

Reply:
Oh no you are right

Reply:
I confused myself